TweetBook/Domain/PostTag.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace TweetBook.Domain
+{
+    public class PostTag
+    {
+       [ForeignKey(nameof(TagName))]
+       public  virtual Tag Tag { get; set; }
```

Comment:
Why are we only declaring the Tag as a foreign key? Isn't the Post also a foreign key?